### DISCUSSION TITLE
Progettazione domotica/impianto elettrico in una casa da ristrutturare

### DISCUSSION THREAD

**Discussion by SaintTDI** [indomus/forum#1655] [GENERALE - Discussioni generiche]
Ciao a tutti,

finalmente la data del rogito della nuova casa si avvicina :) Una volta che abbiamo le chiavi inizieranno i lavori di ristrutturazione completa, ovvero anche impianto elettrico e termoidraulico.

Avviso che sarà un post lungo :D 

Questa è la planimetria della casa di circa 130mq

![image](https://user-images.githubusercontent.com/23402327/164509232-92cf0d56-ec42-4c3b-86a6-00363c27fb62.png)
Cerco di spiegare nel modo migliore come sarà composta casa e quello che pensavo di fare a livello domotico, conosco poco a livello elettrico, sono un informatico :D Penso di aver capito che prodotti Shelly utilizzare, ma vorrei un aiuto da voi per capire se sto scegliendo i prodotti giusti e se la progettazione è fatta bene oppure si può migliorare (direi quasi sicuramente di si :D )

- L'impianto termoidraulico è autonomo, uso già Tado con le valvole termostatiche nella vecchia casa e mi trovo bene, volevo continuare ad usarlo
- L'impianto elettrico di casa sarà sicuramente a 4,5KW, in futuro vederò se portarlo a 6KW ma non credo che serve
- Nel Quadro Elettrico Iniziale arriverà la corrente dal contatore che si trova nel sottoscala, per poi arrivare al Quadro Elettrico Principale, l'elettricista ha detto che bisogna fare questa cosa. **Serve veramente?**
- Nel Quadro Elettrico Principale pensavo di mettere i seguenti magnetotermici, **secondo voi vanno bene oppure sono troppi?**
1. Luci
2. Prese
3. Cucina
4. Frigo
5. Condizionatori
6. Caldaia
7. Luci
8. Bagno 1
9. Bagno 2
- Tutte le tapparelle saranno elettriche
- Ci saranno 4 condizionatori, posizionati in Camera da letto, Studio 1, Studio 2, Salone. Saranno due dual split.
- Il Balcone 2 verrà utilizzato come "stanza tecnica" dove ci saranno i seguenti "elettrodomestici":
1. Caldaia a condensazione
2. Lavatrice
3. Asciugatrice
4. Congelatore
5. I due motori dei 4 condizionatori
- Studio 1 sarà utilizzato da mia moglie per:
1. lavorare in Smart Working (quindi PC portatile, schermo esterno, presa LAN)
2. cucire.
3. Inoltre ci saranno esposti dei Lego in delle vetrine
- Studio 2 sarà utilizzato da me per: 
1. lavorare in Smart Working
2. PC Fisso e qui avrò il modem FTTH, tutti i dispositivi di rete, stampante e qualsiasi altra cosa "informatica", tutto il possibile sotto UPS APC UPS PRO BR900MI da 900VA / 540W che già possiedo. 
3. Anche io avrò parecchie vetrine con Lego esposti :D 
4. Ci sarà una TV
5. Ci sarà un divano letto per eventuali ospiti

- Nel Salone ci sarà:
1. area TV con impianto 5.1 con sintoamplificatore, vari dispositivi come decoder SKY, lettore bluray, retroconsole e via dicendo
2. area tavolo da pranzo 
3. librerie Billy, saranno circa 9 divise in due punti diversi
4. All'ingresso ci sarà un armadio dove mettere le scarpe e cappotti, ci sarà un controsoffitto con dei faretti
- Nel Bagno 1 ci sarà una classica luce a soffitto e una barra led posta sopra al profilo delle mattonelle da rivestimento.
- Nel Balcone grande si potrà mangiare tranquillamente, quindi sarà fornito di un paio di luci a soffitto e un paio di prese per qualsiasi utilizzo
- La cucina avrà le luci al soffitto, varie prese tra il TOP e il pensile per gestire elettrodomestici vari (caffettiera, bimby, bollitore etc...) e ci saranno delle strisce led sotto pensile (non metterò io i Led ma i pensili li compro direttamente con la striscia led incorporata) cosi da poter illuminare il TOP, oltre ovviamente alla lavastoviglie.

Veniamo finalmente a quello che vorrei dal mio impianto domotico ed ai dubbi :D

**So che gli Shelly possono essere usati sia tramite la sua App che tramite Alexa/Google Home. Ma per quanto riguarda l'utilizzo con Siri e l'app "Casa" di Apple come posso fare?**

1. Pensavo di usare il più possibile i prodotti Shelly Pro, cosi da poter utilizzare la connessione con cavo LAN invece che il Wi-Fi, per avere una connessione più stabile ed evitare interferenze. Per fare questo con l'elettricista stavamo pensando di portare direttamente al Quadro Elettrico Principale tutti i cavi delle prese e luci da gestire con i vari Shelly Pro. Farà arrivare un cavo LAN che parte dallo Studio 1, poi vicino al quadro elettrico metterò uno switch di rete per portare la LAN a tutti i vari Shelly PRO
2. Poter accendere e spegnere tutte le luci di casa separatamente ed anche a gruppi. 
a. **Utilizzare X Shelly 4 Pro PM per spegnere luci è giusto? So che non servirebbe il "PM" ma comprare uno Shelly 4 Pro PM costa di meno rispetto a due Shelly 2 Pro**
b. **Per fare dei gruppi immagino che si possa fare tutto tramite l'app giusto?**
c. **Per gestire la striscia Led sotto il pensile della cucina, basta portare anche questa al Quadro Elettrico Principale e collegarla allo Shelly 4 Pro PM giusto?**
d. La striscia led che si trova in bagno, volevo gestirla con uno Shelly Motion. Cosi da accendere la striscia led in automatico quando si entra in bagno, quando fuori si fa buio, e farla spegnere dopo X tempo. La trovo comoda soprattutto di notte :D Il fatto è che non volevo far accendere al massimo della luminosità la striscia Led, ma ad una % molto minore. **Per fare questo mi basta usare uno Shelly Dimmer 2? Non serve collegare quindi questa striscia led ad uno Shelly Pro che si trova sul Quadro Elettrico Principale?**
3. Sapere quanti KW sto consumando a casa, cosi se ci si avvicina troppo ai 4,5KW posso far staccare in automatico qualche dispositivo. 
a. **Mi serve uno Shelly EM per sapere quanto sto consumando giusto? Ne basta uno con una pinza da 50A a monte di tutti i magnetotermici? Oppure è meglio conoscere il carico dei vari magnetotermici? Può essere utile a qualcosa?**
b. **Se volessi staccare il magnetotermico dei condizionatori quando arrivo vicino ai 4,5KW, quale dispositivo Shelly posso usare? Posso usare anche una linea di uno Shelly 4 Pro e quindi staccare tramite APP?**

3. Ovviamente poter alzare ed abbassare tutte le tapparelle di casa. Qui credo che sia inutile portare i cavi delle tapparelle fino al Quadro Elettrico Principale, quindi pensavo di mettere un dispositivo Shelly per ogni 503 dove ci saranno i pulsanti per abbassare ed alzare le tapparelle. **Devo usare lo Shelly Plus 2 PM oppure c'è un dispositivo migliore? Volevo sapere inoltre, come fa lo Shelly a capire fino a quanto aprire o chiudere la tapparella senza sforzare il motore? Il fine corsa lo gestisce il motore oppure qualcos'altro? Inoltre se volessi aprire le tapparelle al 50%, è possibile gestirlo tramite app e quindi Alexa con un comando del tipo "Apri tapparella Cucina al 50%" ?**
4. Poter gestire, quindi accendere e spegnere alcune prese delle varie stanze, anche qui pensavo di usare N Shelly 4 Pro PM.
5. Vorrei comunque avere la possibilità in futuro di poter gestire altre prese, quindi vorrei che nelle varie 503 ci sia lo spazio poi di aggiungere uno Shelly per gestire quella presa. 
6. Poter sapere la temperatura nelle varie stanze, visto che la temperatura che rileva la valvola Tado è troppo vicino al termosifone. Qui potrei usare il Sensore di temperatura con uno Shelly 1 e sensori DS18B20 all'interno di una 503 per conoscere la temperatura giusto?

**In tutto questo gli Shelly si bloccano? Se si, cosa dovrei fare per riavviarli... hanno tutti un pulsante di reset? Facilmente accessibile? L'app va bene oppure ha dei problemi?**

Oltre alle tantissime domande che ho fatto, avete altre idee, miglioramenti a questa progettazione? Cosa fareste voi? Oppure ho scritto una marea de cazzate?! :D 

GRAZIE VERAMENTE TANTE a chi vuole aiutarmi :)

**Comment by dante70** [comment]
ciao
provo a darti la mia opinione su parte di quello chiesto sul quale ho esperienza diretta.
1. I magnetotermici non sono mai troppi nel senso che se hai la possibilita' sia di spazi e budget ecc suddividere nel modo piu' oculato 
    possibile  le utenze ti permette in caso di problemi di isolare il guasto.Per non parlare di eventuale manutenzioni /modifiche future 
    sulle varie linee puoi ben capire che non avrai bisogno di staccare magari tutta una zona o addirittura il generale.
2.  Per le tapparelle ti consiglio di non speculare prima di tutto sui motori in quanto in caso di guasto devi aprire il cassonetto e 
     intervenire materialmente sull'avvolgibile.
     Per esperienza diretta ti posso dire che gli shelly 2.5 vanno molto bene allo scopo.Da febbraio/marzo 2021 ho motorizzato 9 tapparelli 
     con questi apparecchi e stanno funzionando alla perfezione.Io ho messo 7 di questi su una scatola di derivazione nel sottotetto e ho 
    raggrupppato i cavi tutti li.Gli altri 2 sono in una 503 uno e l'altro in una scatola di derivazione.
    Il mio consiglio e' di fare il tutto con anche il pulsante il prossimita' della tapparella(forse e' ovvio) che andra' a comandare lo shelly 
   anche in assenza di wi fi.Se metti ognuno in ogni scatola io proporrei una 504 se devi ristrutturare .....lo spazio non e' mai troppo.
    Gli shelly 2.5 hanno un sensore amprerometrico al loro interno che gli permette di controllare la potenza assorbita dal motore.
   Al momento della taratura siccome i motori e' previsto che si fermino coi finecorsa meccanici al loro interno quando lo faranno lo shelly 
   rilevera' potenza zero e quindi fissera'  il primo punto (apertura ad esempio) .......seguiranno altri movimenti  per determinare anche 
   l'altro punto . Tutto questo in automatico attivando semplicemente da pagina web la calibrazione una volta solo e per sempre.
   Io ho fatto anche  delle card  su HA con tasti che raggruppano varie tapparelle ad esempio: chiudi tutte ,apri tutte chiudi primo 
  piano,apri fronte strada ecc.
 Come si dice in tanti il limite e' la tua fantasia

**Comment by BigMiura** [comment]
Ciao,
provo a darti anche io un parere.

- Come ha detto Dante, i magnetotermici non sono mai troppi, calcola che nella mia casa di 100mq ne ho circa il doppio, tutti i miei utilizzatori di "potenza" sono protetti singolarmente.
- Per le luci io ho utilizzato gli **Shelly 1v3**, preferendo un approccio dell'impianto ad albero piuttosto che il centralizzato come te. Più che altro mi fa pensare la differenza di costo tra _"X"_ **Shelly 1v3** e _"X/4"_ **Shelly 4 Pro PM**. Questi ultimi li utilizzerei solo per le potenze. Comunque con un approccio centralizzato devi portare tutti le luci al QG, quindi anche la striscia led della cucina, attendo solo al voltaggio di quest'ultima.
- L'assorbimento generale lo puoi monitorare con lo **EM**, se fai partire poi le alimentazioni dagli **4 Pro PM** puoi far staccare gli apparecchi a tua scelta configurando dall'App o dal portate Shelly. Tra l'altro, se il funzionamento degli **4 Pro PM** è, come penso, simile agli 1PM puoi impostare le protezioni di massima potenza.

Io uso gli Shelly da quasi un anno, anche tramite Home Assistant, e per ora non ho mai avuto problemi. Un loro qualità che apprezzo tantissimo è la funzionalità in locale, che non blocca l'impianto anche in caso di mancanza di connessione.

Spero di averti aiutato.
Ciao,
Fabio.

**Reply by SaintTDI** [reply]
Ciao,
grazie per le risposte!

@BigMiura il fatto di utilizzare Shelly 4 Pro PM invece dei Shelly 1 o via dicendo... è per poterli collegare con il cavo LAN. Vorrei avere meno dispositivi possibili in Wi-Fi... trovo molto più stabile il cavo LAN :) 
Per la striscia LED sotto la cucina, non credo sia dimmerabile, quindi sarà solo accesa/spenta, e immagino che sarà collegata con il suo alimentatore ad una presa elettrica normale. Quindi dovrò portare il cavo di quella presa alla quadro generale giusto?

Ma per l'uso in locale, quindi anche se manca la connessione, come avviene ? Non riesco a capire come dare l'input allo shelly in locale, tolto il fatto di usare il pulsante al muro :) Perchè usare Alexa serve la connessione ad internet, usare l'app Shelly idem giusto? L'unico utilizzo in locale sarebbe invocare lo shelly da una pagina web locale ... oppure mi sfugge qualcosa?

@dante70 ma per la tapparelle potrei usare lo Shelly Plus 2PM al posto dello Shelly 2.5? Chiedo solamente perchè il plus è un dispositivo più recente, con cpu migliore. Che differenza ci sarebbe ad usare l'uno o l'altro?

grazie ancora!

**Reply by snakuzzo** [reply]
Gli Shelly possono essere pilotati in locale tramite HTTP, MQTT e websocket. Se usi HA e l'integrazione ufficiale, li puoi usare in locale abilitando Coiot dalla pagina web dei settings di ogni shelly (non da app). HA te li troverà in automatico se hai attivo il discovery

**Reply by SaintTDI** [reply]
> Gli Shelly possono essere pilotati in locale tramite HTTP, MQTT e websocket. Se usi HA e l'integrazione ufficiale, li puoi usare in locale abilitando Coiot dalla pagina web dei settings di ogni shelly (non da app). HA te li troverà in automatico se hai attivo il discovery

Ok ti ringrazio :)

**Comment by antony84** [comment]
Ciao, anche io mi trovo in una situazione simile, impianto di casa nuova.
Il dubbio principale è se usare i dispositivi PRO a quadro, centralizzando, o se usate i PLUS nella cassette 50x o nelle scatole di derivazione decentralizzando.

Le questioni diciamo sono 2:
- economica: un pro 4 pm costa il 70% in più di due plus 2 pm (circa 60 euri contro 100), e su 40 gruppi luci come nel mio caso fa la differenza. Oltre al fatto (spesa extra) che vanno portate tutte le luci al QG.
- stabilità e "salubrità" connessione: wifi vs ethernet

Cosa mi consigliate di fare? Vale davvero la pena spendere di più per i PRO?

**Reply by BigMiura** [reply]
Ciao Antony,
premetto che io utilizzo solo dispositivi wifi, e non posso fare confronti con il cablato, ma sinceramente con una buona rete wireless non ho mai avuto problemi. Io ho una rete mesh con 2 "torrette" Xiaomi con 254 dispositivi collegabili, che mi coprono casa e cortile.

Da elettricista personalmente preferisco l'approccio decentrato, con montanti a partire dal QG e cassette di derivazione principali in ogni stanza. Io utilizzerei i PRO a quadro solo per la gestione e il monitoraggio degli utilizzatori di potenza, lavatrice, lavastoviglie, forno, etc; che a casa mia hanno un magnetotermico separato.

La differenza di costo dell'approccio centralizzato per le luci, come scrivi, è notevole e immagino che il QG che dovresti mettere sia più di un 36 moduli, ogni PRO sembrerebbe occupare 3 spazi, più gli interruttori elettrici di base.
A parte il costo degli Shelly la differenza è filosofica, ad un collega "accentratore" non piace mettere le cassette nelle stanze perché alcuni clienti non le vogliono. Io non ho mai avuto il problema, le mie sono nascoste dalle porte o da mobili.

Spero di esserti stato di aiuto.
Ciao,
Fabio.

**Reply by antony84** [reply]
Ciao Fabio, grazie per la risposta.
Gli shelly wifi li hai messi nelle cassette di derivazione o nelle scatole 50x? In quest'ultimo caso un frutto è occupato dallo shelly?

**Reply by BigMiura** [reply]
Ciao Antony,
io li ho messi in entrambi, dove mi era più comodo.
Se le tue porta-frutto non sono molto congestionate ci sta tranquillamente, i moduli sono piccoli. Per esempio, puoi tranquillamente mettere uno Shelly 1 in una 503 con 3 pulsanti.
